src/test/java/duke/data/TaskListTest.java

```diff
@@ -0,0 +1,29 @@
+package duke.data;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.data.task.Todo;
+
+public class TaskListTest {
+    @Test
+    public void testAddTask() {
+        TaskList tasks = new TaskList();
+        tasks.add(new Todo("Do homework"));
+        assertEquals(1, tasks.size());
+        assertEquals("T | 0 | Do homework", tasks.get(0).storageStr());
+    }
+
+    @Test
+    public void testDeleteTask() {
```

Comment:
Same as above! Adding more details will help you remember what the testing method is testing

Reply:
Could you could provide an example?

Reply:
I meant using whatIsBeingTested_descriptionOfTestInputs_expectedOutcome for the naming